dbicdh/_common/upgrade/76-77/001-migrate-audit-events-to-scheduled-products.pl

```diff
@@ -26,11 +26,10 @@
         my $settings;
         try {
             $settings = decode_json($event->event_data);
-        };
-        if (!$settings) {
+        }
+	catch ($e) {
             log_warning(
                 "Unable to read settings from 'iso_create' audit event with ID $event_id. Skipping its migration.");
-            next;
```

Comment:
It's not clear to me why the `next` is not needed anymore

Reply:
I think he removed accidentally, otherwise it will create the scheduled_products and update the event.